apps/nova/src/components/Plans/Content.tsx

```diff
@@ -0,0 +1,210 @@
+        <Tabs.Root
+          value={tab}
+          onValueChange={(value: string) => setTab(value as Tab)}
+        >
+          <Tabs.List className="flex justify-center gap-16">
+            {tabs.map(({ value, label }) => (
+              <Tabs.Trigger
+                key={value}
+                value={value}
+                className={cn("py-2 relative")}
+              >
+                {/* these are quotes of installments above half and annual plans */}
+                {value === "half" ? (
+                  <>
+                    <Typography
+                      element="tiny-text"
+                      weight="regular"
+                      className="absolute -top-[57px] -right-[105px] text-brand-600 bg-natural-50 rounded-[30px] shadow-plan-tooltip p-2"
+                    >
+                      {intl("plans.installments.three-month")}
+                    </Typography>
+                    <div className="absolute -top-[50px] right-1 w-[35px] h-[15px]">
+                      <CurvedDashedLine />
+                    </div>
+                  </>
+                ) : null}
+                {value === "annual" ? (
+                  <>
+                    <Typography
+                      element="tiny-text"
+                      weight="regular"
+                      className="absolute -bottom-[20px] -left-[355%] text-brand-600 bg-natural-50 rounded-[30px] shadow-plan-tooltip p-2"
+                    >
+                      {intl("plans.installments.six-month")}
+                    </Typography>
+                    <div className="absolute -bottom-[35px] -left-[115%] w-[35px] h-[15px] -rotate-[140deg]">
+                      <CurvedDashedLine />
+                    </div>
+                  </>
+                ) : null}
+
+                <Typography
+                  element="body"
+                  weight="semibold"
+                  className={cn(
+                    "transition-colors duration-300",
+                    value === tab ? "text-brand-700" : "text-natural-500"
+                  )}
+                >
+                  {intl(label)}
+                </Typography>
+
+                <AnimatePresence>
+                  {tab === value ? (
+                    <motion.div
+                      initial={{ opacity: 0 }}
+                      animate={{
+                        opacity: 1,
+                        transition: {
+                          duration: 0.3,
+                        },
+                      }}
+                      exit={{ opacity: 0 }}
+                      className="absolute -bottom-[1px] left-0 w-full h-[3px] bg-brand-700 rounded-t-[10px]"
+                    />
+                  ) : null}
+                </AnimatePresence>
+              </Tabs.Trigger>
+            ))}
+          </Tabs.List>
+
+          <div className="mt-8">
+            <AnimatePresence initial={false} mode="wait">
+              {makeTabPlans(tab)}
```

Comment:
The animation will not work as expected this way. You need render all plans inside the `<AnimatePresence/>` to be able to switch between them smoothly. 

Here is an example: 


```suggestion
    {tab === 'monthly'? makeTabPlans('monthly') : null}
    {tab === 'quarter'? makeTabPlans('quarter') : null}
    {tab === 'half'? makeTabPlans('half') : null}
    {tab === 'year'? makeTabPlans('year') : null}
```

this way the `<AnimatePresence/>` will handle the switching between the old component and the new one. It will apply the exist animation on the component you want to remove from the dom then apply the initial animation on the new component.